packages/react-devtools-extensions/webpack.backend.js

```diff
@@ -73,6 +75,7 @@ module.exports = {
       'process.env.DEVTOOLS_PACKAGE': `"react-devtools-extensions"`,
       'process.env.DEVTOOLS_VERSION': `"${DEVTOOLS_VERSION}"`,
       'process.env.GITHUB_URL': `"${GITHUB_URL}"`,
+      'process.env.LOGGING_URL': `"${LOGGING_URL}"`,
```

Comment:
nit: I don't think we need this in the backend config, do we? This variable is only referenced in code imported form `main.js` which is built by the frontend config.